code/game/objects/structures/morgue.dm

```diff
@@ -141,37 +151,91 @@
 		M.set_respawn_bonus("CORPSE_HANDLING", 8 MINUTES)
 
 
+
+
+/obj/structure/morgue/proc/store_atom(var/atom/movable/AM)
```

Comment:
Why `/atom/movable` when it is intended to store only `/obj`?

Reply:
It stores mobs too
Obviously
Like, thats literally its entire function